hathor/p2p/manager.py

```diff
@@ -342,6 +345,7 @@ def on_peer_ready(self, protocol: HathorProtocol) -> None:
         assert protocol.peer is not None
         protocol.peer = self.peer_storage.add_or_merge(protocol.peer)
         assert protocol.peer.id is not None
+        protocol.peer.last_seen = self.reactor.seconds()
```

Comment:
Since `last_seen` is updated on `recv_message` is there any effect updating here too? If there is I think it would be good to leave a comment.

Reply:
You're right. No need. Thanks!

Reply:
Done!